test/package.js

```diff
@@ -24,6 +34,48 @@ test('Every rule is defined in index file', t => {
 		'There are more exported rules than rule files.');
 	t.is(Object.keys(index.configs.recommended.rules).length, ruleFiles.length,
 		'There are more exported rules in the recommended config than rule files.');
+
+	testSorted(t, Object.keys(index.configs.recommended.rules), 'configs.recommended.rules');
```

Comment:
Could also use [`sort-keys`](https://eslint.org/docs/rules/sort-keys) rule.

Reply:
Seems like overkill to setup ESLint to run programmatically just for this one thing.

Reply:
I meant to suggest `/* eslint-enable sort-keys */`

Reply:
Ah ok. Yeah, that would work.